src/app/components/add-class-form/add-class-form.component.html

```diff
@@ -10,6 +10,13 @@
 *ngIf="formIsOpen">
     <h1 class="title">add class</h1>
     <yps-form-field *ngFor="let field of fields" [field]="field"
-    [formControlName]="field.name"></yps-form-field>
+    [formControlName]="field.name">
+    </yps-form-field>
```

Comment:
Don't quite understand for moving [formControlName] to another line, it shoul be on the same line or you can move the other atributes to new lines.